### DISCUSSION TITLE
Does cookiecutter work officially on Python 3.11?

### DISCUSSION THREAD

**Discussion by ZenulAbidin** [cookiecutter/cookiecutter#1840] [Q&A] [answered]
I asked because I could successfully create a cookiecutter project using a Python 3.11 shell on Windows 11 (x64), but the README doesn't list 3.11 as a supported version. Did someone just forget to update the README?

**Comment by ccalvocm** [comment]
Greetings @ZenulAbidin 

Working fine under python 3.11.3

![cookiecutter](https://github.com/cookiecutter/cookiecutter/assets/52244870/606b1a84-898c-4fc1-bf58-b8a2a413f9cb)

**Comment by kurtmckee** [comment, marked as answer]
Python 3.11 is now a tested and supported Python version, as of #1850.
